### DISCUSSION TITLE
primera sesión después de esta semana de inicio?, Es hoy o hasta el próximo lunes

### DISCUSSION THREAD

**Discussion by EMILIO425** [LaunchX-InnovaccionVirtual/CursoIntroPython#250] [Q&A] [answered]
Cuando es la sesión próxima?,

Luego de la semana de prueba

**Comment by DiegoPenalozaJuarez** [comment, marked as answer]
> Cuando es la sesión próxima?,
> 
> Luego de la semana de prueba

Es el dia de hoy compañero, a las 8 pm. Previamente tenias que haber enviado tu formulario con tu repositorio para que te llegue el correo de acceso a la siguiente misión.

**Reply by Daniel-Cruz19** [reply]
Que tal compañero, la siguiente clase cuando es?, aun no acabo las katas y se que la primer clase fue hoy, me podrías decir cuando es la siguiente por favor para ponerme al corriente lo mas antes posible, gracias.

**Reply by Ingrzs** [reply]
El miércoles alas 7 pm
